src/main/kotlin/com/grepp/nbe1_3_team04/team/service/request/TeamDefaultServiceRequest.kt

```diff
@@ -0,0 +1,7 @@
+package com.grepp.nbe1_3_team04.team.service.request
+
+data class TeamDefaultServiceRequest(
+    val name: String,
+    val description: String,
+    val location: String
```

Comment:
```suggestion
    val description: String?,
    val location: String?
```
여길 이렇게 바꿔주시고 api request 단에서 default 값을 지정을 안해주는게 좋을것같습니다!